src/binaryen-c.cpp

```diff
@@ -104,6 +104,7 @@ BinaryenType BinaryenInt32(void) { return i32; }
 BinaryenType BinaryenInt64(void) { return i64; }
 BinaryenType BinaryenFloat32(void) { return f32; }
 BinaryenType BinaryenFloat64(void) { return f64; }
+BinaryenType BinaryenUnreachableType(void) { return unreachable; }
```

Comment:
This makes me sad.

Reply:
Yeah...

What do you think about changing these to `BinaryenTypeInt32`, `BinaryenTypeUnreachable` etc.? We could keep the old ones for backwards compatibility, but I think we can get away with frequent breaking changes, currently.